src/Our.Umbraco.Ditto.Contrib/Collections/PublishedContentFilter.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Umbraco.Core.Models;
+using Umbraco.Web;
+
+namespace Our.Umbraco.Ditto
+{
+    public abstract class PublishedContentFilter : DittoProcessorAttribute
```

Comment:
Can't help but wonder if we could make a generic Filter attribute that can just filter any collection?